optimize explorer (#1365)
vesoft-inc/nebula-docs: docs-2.0/nebula-explorer/12.query-visually.md

| nGQL                                                         | Click **nGQL** to view the statement corresponding to the query pattern.                             |
| Run Query                                                         | Click **Run Query** to display the query results visually on the canvas.                 |

## Steps

> 1. Drag several target tags from on the left side of the **Visual Query** page to create the corresponding vertexes on the right side.

Reviewer: ```suggestion
1. Drag several target tags from the left side of the **Visual Query** page to the canvas to create the corresponding vertices.
```

2. Click a vertex, hold down the left mouse button on the anchor point at the edge of the vertex, and drag it to another vertex to create the corresponding edge.

> 3. Click a vertex to configure configurations. The description are as follows.

Reviewer: ```suggestion
3. Set a vertex by clicking it. The descriptions of configuration options are as follows.
```

  ![vertex_confi](https://docs-cdn.nebula-graph.com.cn/figures/vertex-config_en.png)

  - **Tag Name**: Set zero, one, or multiple tags.

  !!! note

        One vertex can have zero or multiple tags:

        - When 0 tag is set, query the vertex without tags.
        - When 1 tag is set, query the vertex with that tag.
        - When multiple tags are set, query the vertex that has all the tags you set. 

  - **Filter**: Add one or more sets of filter conditions, including vertex properties, operators, and property values. 

  !!! enterpriseonly

        When setting multiple tags in the **Tag Name** dialog box, only the Nebula Graph Enterprise Edition supports setting **filter conditions** to query data.

> 4.  Click a edge to configure configurations. The description are as follows.

Reviewer: ```suggestion
4.  Set an edge by clicking. The descriptions configuration options are as follows.
```

  ![vertex_confi](https://docs-cdn.nebula-graph.com.cn/figures/edge-config_en.png)

  - **Edge Type**: Set one or multiple edge types.

  !!! note

        One edge have one and only one edge type:

        - When one edge type is set, query the edge with that edge type.

        - When multiple edge types are set, query the edge that has any of the edge types you set.

  - **Direction**: Set the edge direction between two vertices, including Outgoing, Incoming, and Bidirect.

  - **Single**: Set a fixed-length path.

  - **Range**: Set a variable-length.

  - **Filter**: Add one or more sets of filter conditions, including edge properties, operators, and property values.

  !!! enterpriseonly

        When setting multiple edge types in the **Edge Type** dialog box, only the Nebula Graph Enterprise Edition supports setting **filter conditions** to query data.

5. After the query scenarios (pattern) is created, click ![Output](https://docs-cdn.nebula-graph.com.cn/figures/visual-nav-output.png) and select the result you want to return.

> 6. Click **Run Query** on the upper right corner of the **Visual Query** page to display the query results in the canvas.

Reviewer: ```suggestion
6. Click **Run Query** on the upper right corner of the **Visual Query** page to display the query results on the canvas.
```

2. Configure filter conditions.

   1. Set the filter condition for the first player to `player.name == Yao Ming`.

>    2. Set the edge type of the edge between the first vertex with tag player and second players to `follow`, set the direction to `Bidirect`, and the steps to `1`.

Reviewer: ```suggestion
   2. Set the edge type of the edge between the first and second vertices of the tag player to `follow`, set the direction to `Bidirect`, and the steps to `1`.
```
